docs/readthedocs/source/doc/Nano/Overview/tensorflow_inference.md

```diff
@@ -1,98 +1,225 @@
 
 # (Optional) step 3: Something else, like training ...
 model.fit(train_dataset)
 
-# step 4: execute quantization
-q_model = model.quantize(calib_dataset=train_dataset)
+# step 4: execute quantization using `OpenVINO` acceleration
+traced_model = InferenceOptimizer.trace(model, accelerator="openvino")
+# or setp 4: execute quantization using `ONNXRuntime` acceleration
+traced_model = InferenceOptimizer.trace(model, accelerator="onnxruntime")
 
 # run simple prediction
-y_hat = q_model(train_examples)
+y_hat = traced_model(train_examples)
 
-# evaluate, predict also support acceleration
-q_model.evaluate(train_dataset)
+# predict also support acceleration
+traced_model.predict(train_dataset)
+```
+
+## Quantization
+
+Quantization is widely used to compress models to a lower precision, which not only reduces the model size but also accelerates inference. BigDL-Nano provides `InferenceOptimizer.quantize()` API for users to quickly obtain a quantized model with accuracy control by specifying a few arguments. `Sequential` has similar usage, so we will only show how to use an instance of `Model` to enable quantization pipeline here.
+
+To use INC as your quantization engine, you can choose `accelerator=None/'onnxruntime'`. Otherwise, `accelerator='openvino'` means using OpenVINO POT (Post-training Optimization) to do quantization.
+
+### Quantization without extra accelerator
+
+By default, `InferenceOptimizer.quantize()` doesn't search the tuning space and returns the fully-quantized model without considering the accuracy drop. If you need to search quantization tuning space for a model with accuracy control, you'll have to specify a few arguments to define the tuning space. More instructions in [Quantization with Accuracy Control](#quantization-with-accuracy-control)
+
+Taking the example in [Runtime Acceleration](#runtime-acceleration), you can use quantization as following:
+
+```python
+# use Intel Neural Compressor quantization
+q_model = InferenceOptimizer.quantize(model, x=train_dataset)
+
+# or use ONNXRuntime quantization
+q_model = InferenceOptimizer.quantize(model, x=train_dataset, accelerator="onnxruntime")
+
+# or use OpenVINO quantization
+q_model = InferenceOptimizer.quantize(model, x=train_dataset, accelerator="openvino")
+
+# you can also use features and labels instead of dataset for quantization
+q_model = InferenceOptimizer.quantize(model, x=train_examples, y=train_labels)
+
+# you can use quantized model as before
+y_hat = q_model(train_examples)
 q_model.predict(train_dataset)
 ```
+
 This is a most basic usage to quantize a model with defaults, INT8 precision, and without search tuning space to control accuracy drop.
 
-To use quantization, you must use functional API to create a keras model. This is a known limitation
-of INC.
+```eval_rst
+.. note::
+    To use quantization, you must use functional API to create a keras model. This is a known limitation of INC.
```

Comment:
Does it mean here that for quantization using INC, we can only use `keras.Model` instead of `keras.Sequential`? Does that mean we could still use POT quantization for `keras.Sequential`?

If so, maybe we should refactor this sentence a little bit, so that we could emphasize that `keras.Sequential` can still be used for POT quantization.